common/services/catalogue/routes.js

```diff
@@ -3,6 +3,26 @@ import { type NextLinkType } from '@weco/common/model/next-link-type';
 
 type Params = { [key: string]: any };
 export type UrlParams = { [key: string]: string };
+
+export function serialiseQueryParams(
```

Comment:
I'd move this into a util method as it isn't related to our routes.

Reply:
sounds good